mobile/envoy_build_config/BUILD

```diff
@@ -14,6 +14,7 @@ envoy_cc_library(
     deps = [
         "extension_registry_platform_additions",
         "@envoy//source/common/network:socket_lib",
+        "@envoy//source/common/quic:quic_transport_socket_factory_lib",
```

Comment:
This may be missing envoy_select_enable_http3 usage https://github.com/envoyproxy/envoy/pull/24496/files#diff-4517a4a52636c1c8e390c1d9653aa000a67fc1070d977ebc114ca9eacac2ec49R48